src/EditorFeatures/Test2/Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.vbproj

```diff
@@ -16,7 +16,11 @@
     <ProjectReference Include="..\..\Compilers\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.csproj" />
     <ProjectReference Include="..\..\Compilers\VisualBasic\Portable\Microsoft.CodeAnalysis.VisualBasic.vbproj" />
     <ProjectReference Include="..\..\Interactive\EditorFeatures\Core\Microsoft.CodeAnalysis.InteractiveEditorFeatures.csproj" />
+    <ProjectReference Include="..\..\Interactive\Host\InteractiveHost32.csproj" />
     <ProjectReference Include="..\..\Test\Utilities\Portable\Roslyn.Test.Utilities.csproj" />
+    <ProjectReference Include="..\..\VisualStudio\Core\SolutionExplorerShim\Microsoft.VisualStudio.LanguageServices.SolutionExplorer.csproj" />
```

Comment:
What required this? I was about to block for this but I see editor-features-tests-depending-on-VS-layer is already happening. Most bad, but not your fault. What caused the change though here to let more cats out of more bags?

Reply:
➡️ Tracking this request under https://github.com/dotnet/roslyn/pull/28781#discussion_r208375303.